x/ibc/02-client/exported/exported.go

```diff
@@ -215,3 +217,61 @@ func ClientTypeFromString(clientType string) ClientType {
 		return 0
 	}
 }
+
+// Height is an interface for a monotonically increasing data type
+// that can be compared against for the purposes of updating and freezing clients
+// Compare implements a method to compare two heights. When comparing two heights a, b
+// we can call a.Compare(b) which will return
+// negative if a < b
+// zero     if a = b
+// positive if a > b
+//
+// Decrement will return a decremented height from the given height. If this is not possible,
+// an error is returned
+// Valid returns true if height is valid, false otherwise
+type Height struct {
+	EpochNumber uint64 `json:"epoch_number" yaml:"epoch_number"`
+	EpochHeight uint64 `json:"epoch_height" yaml:"epoch_height"`
+}
+
+func NewHeight(epochNumber, epochHeight uint64) Height {
+	return Height{
+		EpochNumber: epochNumber,
+		EpochHeight: epochHeight,
+	}
+}
+
+// Compare implements clientexported.Height
+// It first compares based on epoch numbers, whichever has the higher epoch number is the higher height
+// If epoch number is the same, then the epoch height is compared
+func (h Height) Compare(other Height) int64 {
```

Comment:
Currently the compare function returns an `int64` to use as comparison.

With negative indicating LT, zero indicating EQ, and positive indicating `GT`.

I could change this to an enum if desired.

Reply:
Whatever is most idiomatic in Go is fine with me

Reply:
I think we should only use -1, 0, and 1 for the return values. That's the standard on other Go libraries